httpclient5/src/main/java/org/apache/hc/client5/http/protocol/RequestAddCookies.java

```diff
@@ -78,7 +79,7 @@ public void process(final HttpRequest request, final EntityDetails entity, final
         Args.notNull(context, "HTTP context");
 
         final String method = request.getMethod();
-        if (method.equalsIgnoreCase("CONNECT") || method.equalsIgnoreCase("TRACE")) {
+        if (method.equalsIgnoreCase(Method.CONNECT.name()) || method.equalsIgnoreCase(Method.TRACE.name())) {
```

Comment:
@arturobernalg Could you please use `Method#isSame` instead of `String#equalsIgnoreCase`?

Reply:
Yep, changed